cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -162,15 +168,18 @@ public String cruiseControlHost(String clusterName, String clusterNamespace) {
                 KafkaRebalance,
                 KafkaRebalanceList> mockRebalanceOps = supplier.kafkaRebalanceOperator;
 
+        when(mockCmOps.getAsync(eq(CLUSTER_NAMESPACE), eq(RESOURCE_NAME))).thenReturn(Future.succeededFuture(new ConfigMap()));
```

Comment:
Do you need the `eq()`?

Reply:
I have tried with both with and without eq(), looks like we dont require it since both leads the same result